doc/tutorials/docbuild.rst

```diff
@@ -150,14 +153,26 @@ Depending on your Linux version, install the needed tools:
 
      sudo dnf install doxygen python3-pip python3-wheel make graphviz
 
-And for either Linux environment, install the remaining python-based
+And for any of these Linux environments, install the remaining python-based
 tools:
 
 .. code-block:: bash
 
    cd ~/projectacrn/acrn-hypervisor/doc
    pip3 install --user -r scripts/requirements.txt
 
+Add ``$HOME/.local/bin`` to your ``PATH`` so the system will find the
+``sphinx-build`` utility:
```

Comment:
change to

    expected versions of python utilities such as ``sphinx-build`` and ``breathe``:

Reply:
Done